src/sentry/static/sentry/app/api.tsx

```diff
@@ -26,12 +29,35 @@ export class Request {
   }
 }
 
+type ParamsType = {
+  itemIds?: Array<number>;
+  query?: string;
+  environment?: string | null;
+  project?: Array<number> | null;
+};
+
+type QueryArgs =
+  | {
```

Comment:
Should there be a leading `|` here?

Reply:
I tried to remove it; but prettier seems to be adding this leading `|`.